docs/articles/samples/IntroCustomEnvironmentInfo.md

```diff
@@ -0,0 +1,39 @@
+---
+uid: BenchmarkDotNet.Samples.IntroCustomEnvironmentInfo
+---
+
+## Sample: IntroCustomEnvironmentInfo
+
+You can add any useful information about environment to the benchmark report. Just mark one or several static methods that return `string` or `IEnumerable<string>` by the `[CustomEnvironmentInfo]` attribute.
+
+### Source code
+
+[!code-csharp[IntroCustomEnvironmentInfo.cs](../../../samples/BenchmarkDotNet.Samples/IntroCustomEnvironmentInfo.cs)]
+
+### Output
+
+```markdown
+BenchmarkDotNet=v0.10.x-mock, OS=Microsoft Windows NT 10.0.x.mock, VM=Hyper-V
+MockIntel Core i7-6700HQ CPU 2.60GHz (Max: 3.10GHz), 1 CPU, 8 logical and 4 physical cores
+Frequency=2531248 Hz, Resolution=395.0620 ns, Timer=TSC
+Single line
```

Comment:
It would be better to actually print some information about the environment: it will help to understand how this feature can be used in real-world scenarios.